Project.toml

```diff
@@ -1,9 +1,10 @@
 name = "DynamicExpressions"
 uuid = "a40a106e-89c9-4ca8-8020-a735e8728b6b"
 authors = ["MilesCranmer <miles.cranmer@gmail.com>"]
-version = "0.16.0"
+version = "0.17.0"
```

Comment:
Maybe a nice moment for a 1.0.0? 😄 1.0.0 allows for specifying new breaking releases x.0.0, feature releases 1.x.0, and bug releases 1.0.x. Also, it makes the package look more trustworthy, which I think is fair given your focus on building quality software, and biting the bullet has to happen one day anyway (or end up at https://0ver.org/).

Reply:
You’re not wrong… 😅

I’m going to do PySR and SymbolicRegression 1.0 soon. I just want to be confident the API has stabilised first (which I guess it has!)

Reply:
Actually since I'm just introducing the Expression type now, I do want to wait a few more minor releases before a 1.0, to accommodate any short-term breaking changes. But SR.jl and PySR perhaps sooner.

Reply:
I get that. It's always a tough call to make. Thanks for considering it!